fedmsg_fasclient.py

```diff
@@ -89,8 +89,18 @@ def delayed_consume():
     def action(self, messages):
         self.log.debug("Acting on %s" % pprint.pformat(messages))
 
+        playbook = 'run_fasClient_simple.yml'
+        groups = [
+            msg['msg']['group']
+            for msg in messages
+            if 'sysadmin-' in msg['msg'].get('group', None)
```

Comment:
I was wondering if there are groups having ``sysadmin`` in their name but not related to us (I'm thinking a ``foosysadmingit`` or so.